04_HowTos/QueryClient/How_to_use_the_Data_Lab_QueryClient.ipynb

```diff
@@ -40,36 +45,52 @@
     "\n",
     "#### Output formats\n",
-    "The results can be returned as whitespace delimited (<i>ascii</i>), CSV (<i>csv</i>), FITS object (<i>fits</i>), HDF5 (<i>hdf5</i>, or in VOTable format (<i>votable</i>). Note that if the results are saved to the user's database then the output format is ignored.\n",
+    "The results can be returned as whitespace delimited (*ascii*), CSV (*csv*), FITS object (*fits*), HDF5 (*hdf5*, or in VOTable format (*votable*). Note that if the results are saved to the user's database then the output format is ignored.\n",
+    "\n",
+    "#### Saving results\n",
+    "If no save location is specified (no *out* param) then the results are returned directly. A save location beginning with the 'vos://' identifier indicates a location in the user's virtual storage to save the result. A save location beginning with the 'mydb://' identifier indicates the results are to be saved to a table in the user's remote database (MyDB). \n",
     "\n",
     "#### Asynchronous queries\n",
-    "Long queries should be run asynchronously and the service may refuse any synchronous query attempted if the projected query time is too long. A query can be submitted asynchronously by setting the <i>async_</i> parameter to <i>True</i>. A job id will then be returned.\n",
+    "Long queries should be run asynchronously and the service may refuse any synchronous query attempted if the projected query time is too long. A query can be submitted asynchronously by setting the *async_* parameter to *True*. A job id will then be returned.\n",
     "\n",
-    "The status of an asynchronous query can be checked by submitting an HTTP GET call to the query manager service <i>status</i> endpoint with the relevant job id as an argument: /status?jobid=<jobid>. A return value of 'COMPLETED' indicates the query has terminated. A return value of 'ERROR' indicates that there was a problem with the query.\n",
+    "    \n",
+    "The status of an asynchronous query can be checked with *queryClient.status(jobid)*. A return value of 'COMPLETED' indicates the query has terminated. A return value of 'ERROR' indicates that there was a problem with the query.    \n",
     "\n",
-    "The results of an asynchronous query (assuming that they were not saved to either the user's virtual storage or remote database) can be retrieved once the query has completed with an HTTP GET call to the query manager service <i>results</i> endpoint with the relevant job id as argument: /results?jobid=<jobid>\n",
+    "The results of an asynchronous query (assuming that they were not saved to either the user's virtual storage or remote database) can be retrieved once the query has completed with *result = qc.query(adql=query,async_=True,wait=True,poll=1,verbose=1)*\n",
```

Comment:
I propose to let this be two cases, both described:

1) a manual retrieval using a jobid:

jobid = qc.query(sql=query,async+=True)
print(jobid)
print(qc.status(jobid)) # repeat until 'COMPLETED'
result = qc.results(jobid)

2) A mechanism to run an async query but to make the client auto-loop until the result is available:

result = qc.query(adql=query,async_=True,wait=True,poll=1,verbose=1)

Only case 2) should be coded in the notebook. (but case 1 can be linked to in the 
01_GettingStartedWithDataLab/01_JupyterPythonSQL101.ipynb NB, or even repeated here).